src/main/java/com/github/fishio/EnemyFish.java

```diff
@@ -52,8 +52,17 @@ public void render(GraphicsContext gc) {
 		if (isDead()) {
 			return;
 		}
+		
+		//TODO Move this to game thread.
```

Comment:
Let's try to avoid TODO's at this point of the project, at least create an issue with it so it can be fixed in a sprint.

<review, organisation